assets/src/infrastructure/ui/layout/modal/modalCloseButtonProps/styles.scss

```diff
@@ -0,0 +1,17 @@
+// this selector is used for specificity
+[role='alertdialog'] {
+	.confirm-close {
+		display: inline-flex;
+		position: absolute;
+		top: 8px;
+		right: 12px;
+		min-width: auto;
+		width: var(--ee-padding-big);
+		height: var(--ee-padding-big);
+		svg {
+			height: var(--ee-border-radius-big);
+			margin: 0;
+			width: var(--ee-border-radius-big);
+		}
```

Comment:
I like this better: 

![image](https://user-images.githubusercontent.com/1477453/79575817-06431f00-80cb-11ea-80fd-09a4ae39f656.png)

Reply:
I needed `--ee-border-radius-big` to match `12px` which is the same dimension as in chakra.

Reply:
Open to better variable names than `--ee-border-radius-big`

Reply:
ya I'll most likely tweak things once everything is converted, but that doesn't look bad as is

Reply:
thanks